packages/source-html/src/translator.ts

```diff
@@ -1,39 +1,45 @@
-import atjsonSchema from '@atjson/schema';
 import Document from '@atjson/document';
+import schema from '@atjson/schema';
```

Comment:
This doesn't use `source-html`'s schema?

Reply:
This is the translator to our common format; we maaaay remove this at a certain point so we can provide different "layers" of annotations.